torchaudio/models/conv_tasnet.py

```diff
@@ -102,6 +102,7 @@ def __init__(
         num_hidden: int,
         num_layers: int,
         num_stacks: int,
+        msk_activate: str,
```

Comment:
how about calling this `mask_activate` to match the attribute name below and adding a comment to the args list above?

Reply:
This aligns with other ``msk_xx`` arguments. If we replace all arguments then it'll be BC-Breaking. Let's fix it so far and make another PR for renaming.